idb/postgres/internal/encoding/encode.go

```diff
@@ -53,7 +54,39 @@ func convertEvalDelta(evalDelta types.EvalDelta) types.EvalDelta {
 	return evalDelta
 }
 
+func desanitizeNull(str string) string {
```

Comment:
I think, we shouldn't desanitize. Essentially, we need to choose between the zero byte and `\u0000` in the string. If we don't desanitize, then the zero byte suffers, if we do desanitize, then `\u0000` suffers. I would prefer that the zero byte suffers.